src/Helpers/Platform/PlatformHelper.js

```diff
@@ -11,11 +11,13 @@ export function getPlatform(platformId) {
   return api.showSource(platformId);
 }
 
-export function getPlatformItems(apiProps) {
+export function getPlatformItems(apiProps, options) {
   let apiPromise = null;
 
   if (apiProps) {
-    apiPromise = fetch(`${TOPOLOGICAL_INVENTORY_API_BASE}/sources/${apiProps}/service_offerings?archived_at=`).then(data => data.json());
+    apiPromise = fetch(`${TOPOLOGICAL_INVENTORY_API_BASE}/sources/${apiProps}/service_offerings?archived_at=${options
```

Comment:
@Hyperkid123 Is this making a direct URL call instead of using the API?

Reply:
@mkanoor Yes, it has been discussed and the api does not support filtering based on `archived_at` at the moment so we had to take this approach for now.

Reply:
@mkanoor if i use the client method i cannot add anything but the pagination options to the request. I expect this to be resolved by the API some time in the future.